terraform/repositories.tf

```diff
@@ -24,6 +24,9 @@ locals {
     },
     "callisto-ui-nginx" : {
       "checks" : local.checks.drone
+    },
+    "callisto-timecard-restapi" : {
+      "checks" : local.checks.drone
     }
```

Comment:
Does this need a comma?

Reply:
Oh weird I didn’t realise I’d missed it but TIL:

> Commas are required between key/value pairs for single line maps. A newline between key/value pairs is sufficient in multi-line maps
https://www.terraform.io/language/configuration-0-11/syntax

So apparently not but I’ll put it in just for consistency 🤷 Good spot 👍